src/main/js/controls/Gantt/helpers/creationHelpers.js

```diff
@@ -291,11 +291,22 @@ const getYAxisRange = (trackList) =>
  * @returns {undefined} - returns nothing
  */
 const scaleGraph = (scale, config) => {
-    scale.x = d3.time
-        .scale()
-        .domain(config.axis.x.domain)
-        .range([0, getXAxisWidth(config)])
-        .clamp(true);
+    if (
+        config.pan !== undefined &&
```

Comment:
I feel like we talked about this before didnt we? 
We need to create a dictionary of settings that we are changing when panning is enabled. https://github.com/cerner/carbon-graphs/issues/41#issuecomment-510461080.

Reply:
regarding dictionary of setting we didn't had any discussion only how the pan object should look like we discussed.
```
pan: {
    enabled: true,
    showControls: true
}
```
i will refer the #41 (comment) for this.